Generalize the mode option in pre-commit (#3765)
demisto/demisto-sdk: demisto_sdk/commands/pre_commit/README.md

### Automatically, as a git hook
* Create a [git hook](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) that calls `demisto-sdk pre-commit`.

## Modes

> If you need different args set for the different modes, for example some rules should be excluded in the nightly build.

Reviewer: ```suggestion
When different args set for the different modes are needed, for example, some rules should be excluded in the nightly build.
```
